types/passport-deezer/tslint.json

```diff
@@ -0,0 +1,6 @@
+{ 
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+      "npm-naming": [true,{"mode":"code","errors":[["NeedsExportEquals",false]]}]
+    }
```

Comment:
remove these and fix errors

Reply:
ok I removed it and used namespace again because i'm getting the following `npm-naming` error while testing: `The declaration should use 'export =' syntax because the JavaScript source uses 'module.exports =' syntax...`